api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/BFFApiGatewayController.java

```diff
@@ -602,12 +602,20 @@ public Flux<ProductResponseDTO> getProductsInInventoryByInventoryIdAndFields(@Pa
         return inventoryServiceClient.getProductsInInventoryByInventoryIdAndProductsField(inventoryId, productName, productPrice, productQuantity);
     }
 
-
+    @GetMapping(value = "inventory")
+    public Flux<InventoryResponseDTO> searchInventory(@RequestParam(required = false) String inventoryName,
+                                                      @RequestParam(required = false) String inventoryType,
+                                                      @RequestParam(required = false) String inventoryDescription){
+        return inventoryServiceClient.searchInventory(inventoryName, inventoryType, inventoryDescription);
+    }
+    /*
```

Comment:
Can you remove the commented-out code brother? Everything looks good other than that great job Matthew.

Reply:
Thank you for your comment I left it there in case we might need it in the future. If the team agrees that is is unnecessary and that we don't need it. I will make sure it is removed.

Reply:
I agree with you matthew, however next time you should delete uncommnented code.

Reply:
Great thinking Matthew, but I feel like removing the comments will clean the code since we will not be needing it.

Reply:
I agree with you Elias, removing the comments will make the code more clear.

Reply:
In my next pull request, I'll remove it, just in case I'm still using it. Thanks to the team for the feedback; you've really helped me become a better developer.